secrets

```diff
@@ -0,0 +1,3 @@
+AWS_KEY=AKIA1835sb
+password="ABC123DOREMI"
```

Comment:
### :warning: Aqua detected sensitive data in your code  
**Category:** Code Generic   
**Description:** Code Generic Credential   
**Severity:** HIGH   
**Match:** password="************"  
[This comment was created by Aqua Pipeline]